pvt/common/skipFoundryTests.ts

```diff
@@ -0,0 +1,6 @@
+import { subtask } from 'hardhat/config';
+import { TASK_COMPILE_SOLIDITY_GET_SOURCE_PATHS } from 'hardhat/builtin-tasks/task-names';
+
+subtask(TASK_COMPILE_SOLIDITY_GET_SOURCE_PATHS).setAction(async (_, __, runSuper) =>
```

Comment:
You'll need to import this file in the base hardhat config

Reply:
Done.

Just as a sidenote, yarn / hardhat doesn't pick up these files either way if we put them inside `test/foundry`. Anyway I think it's good to have this filter.